assets/js/src/form_editor/blocks/submit/edit.jsx

```diff
@@ -37,7 +37,7 @@ const SubmitEdit = ({ attributes, setAttributes }) => {
 
   const styles = !attributes.styles.inheritFromTheme ? {
     fontWeight: attributes.styles.bold ? 'bold' : 'inherit',
-    borderRadius: attributes.styles.borderRadius !== undefined ? `${attributes.styles.borderRadius}px` : 1,
+    borderRadius: attributes.styles.borderRadius !== undefined ? `${attributes.styles.borderRadius}px` : 0,
```

Comment:
With this change, you reintroduced the inconsistency (0 is rendered but 1 is displayed in settings) in settings and rendering. We need to update [the value in submit settings component](https://github.com/mailpoet/mailpoet/blob/master/assets/js/src/form_editor/blocks/submit/styles_settings.tsx#L94) to use zero as well.